configure.ac

```diff
@@ -1,4 +1,5 @@
-AC_INIT([logrotate],[3.11.0])
+AC_INIT([logrotate],[m4_esyscmd([build-aux/git-version-gen .tarball-version])],
+[logrotate-owner@fedoraproject.org],[],[https://github.com/logrotate/logrotate])
```

Comment:
Not that I know of.  Since logrotate upstream was moved to Github, there is probably no "native" way to have a mailing list for the upstream project ... and so far nobody has complained that we do not have such a mailing list.